gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/TimeAggregationsWithGroupByRequestHandler.java

```diff
@@ -304,4 +306,33 @@ private Map<String, AttributeMetadata> remapAttributeMetadataByResultName(
             .collect(Collectors.toUnmodifiableList()),
         attributeMetadataByIdMap);
   }
+
+  private void updateExploreResponse(ExploreResponse.Builder timeAggregationsResponse) {
```

Comment:
Isn't this changing the existing output behaviour? So, we never returned the `null` group?